packages/smart-wallet/src/proposals/upgrade-walletFactory-proposal.js

```diff
@@ -0,0 +1,252 @@
+// @ts-check
+import { E } from '@endo/far';
+import { makeMarshal } from '@endo/marshal';
+
+console.warn('@@@ upgrade-walletFactory-proposal.js module evaluating');
```

Comment:
how much logging is appropriate here?

I suppose one answer is: as much as it took to get the thing working.

Reply:
I mostly agree. While figuring out what's going on, it may take more logging than would be necessary to diagnose problems in the future. But proposals are critical operations that run once, so I think it's appropriate to error towards verbosity